base/multimedia.jl

```diff
@@ -94,9 +94,12 @@ your images to be displayed on any PNG-capable `AbstractDisplay` (such as IJulia
 to `import Base.show` in order to add new methods to the built-in Julia function
 `show`.
 
-The default MIME type is `MIME"text/plain"`. There is a fallback definition for `text/plain`
```

Comment:
Why remove the information about the fallback? That's quite useful to understand how the whole system works.